src/table/head/table-head.component.ts

```diff
@@ -24,7 +26,8 @@ import { Observable } from "rxjs";
 	template: `
 	<ng-container *ngIf="model">
 		<tr>
-			<th ibmTableHeadExpand *ngIf="model.hasExpandableRows()"></th>
+			<th ibmTableHeadExpand *ngIf="model.hasExpandableRows() && !stickyHeader"></th>
+			<th ibmTableHeadExpand *ngIf="model.hasExpandableRows() && stickyHeader" class="bx--table-expand-v2" style="padding-left: 2.5rem"></th>
```

Comment:
Could the class be applied conditionally with an `ngClass`? Also We may want to move the `style` to the `styles` property on the component if we need it (what does react do for that case?)

Reply:
Their expanded for does this:
![Screen Shot 2019-11-25 at 1 49 48 PM](https://user-images.githubusercontent.com/45505172/69568883-7caf7b00-0f8a-11ea-8748-834d8d29147e.png)